README.md

```diff
@@ -2,6 +2,13 @@
 
 The goal of *starfish* is to **prototype** a reference pipeline for the analysis of image-based transcriptomics data that works across technologies. This is a **work in progress** and will be developed in the open.
 
+We are currently in **pre-alpha**, finishing proof of concept pipelines for each of the spaceTx contributors that leverage starfish's shared object model.
+At this time starfish is mature enough to support computational developers interested in adapting other assays to starfish's object model.
```

Comment:
What other assays? Do you mean, assays not implemented by the spaceTx community? If so, I think we should make a table of assays that we currently support, and ones that we intend to support. Perhaps this is in your roadmap, but I believe it belongs in here as well.